packages/dialtone-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dialpad/dialtone-icons",
-  "version": "4.15.0",
+  "version": "4.15.1-alpha.1",
```

Comment:
alpha version set here

Reply:
Shouldn't be an issue as with the next release it'd be overwritten. I can try cleaning it up if needed.